pymc/backends/arviz.py

```diff
@@ -478,6 +478,7 @@ def is_data(name, var) -> bool:
                 and var not in self.model.observed_RVs
                 and var not in self.model.free_RVs
                 and var not in self.model.potentials
+                and var not in self.model.value_vars
```

Comment:
We should have a direct test for `is_data` although that is arguably outside of the scope of this PR

Reply:
I remember this part affected sampling from variational posterior so I was forced to changed it

Reply:
Yeah, what I mean is we should create a separate test just to test this function behavior. Can you open a reminder issue for this?